docs/sources/configure/_index.md

```diff
@@ -3624,7 +3624,19 @@ backoff_config:
   # CLI flag: -<prefix>.backoff-retries
   [max_retries: <int> | default = 10]
 
-# Enable TLS in the GRPC client. This flag needs to be enabled when any other
+# Initial stream window size. Values less than the default are not supported and
+# are ignored. Setting this to a value other than the default disables the BDP
+# estimator.
+# CLI flag: -<prefix>.initial-stream-window-size
+[initial_stream_window_size: <int> | default = 63KiB1023B]
```

Comment:
:sob: 
 
```suggestion
[initial_stream_window_size: <int> | default = 64KiB]
```

Reply:
// The default value of flow control window size in HTTP2 spec.
	defaultWindowSize = 65535

Reply:
it looks like it's not a mistake

Reply:
make format